kratos/python_scripts/time_based_ascii_file_writer_utility.py

```diff
@@ -16,6 +16,8 @@ def __init__(self, model_part, params, file_header):
 
         default_settings = KratosMultiphysics.Parameters('''{
             "output_file_name"  : "",
+            "save_in_folder"    : true,
+            "output_folder"     : "TimeBasedAsciiResults",
```

Comment:
Can you rename it to "output_folder_name"?
=> then it would be consistent with "output_file_name"

Reply:
Correct, modified.